clients/src/main/java/org/apache/kafka/clients/consumer/internals/Coordinator.java

```diff
@@ -317,7 +317,7 @@ public void handle(JoinGroupResponse joinResponse, RequestFuture<Void> future) {
                 log.debug("Joined group: {}", joinResponse.toStruct());
 
                 // record re-assignment time
-                sensors.partitionReassignments.record(response.requestLatencyMs());
```

Comment:
Sure +1 for consistency. I would vote in favor of suffix ending with Ms, since it indicates milliseconds.